R/provider-azure.R

```diff
@@ -214,6 +214,19 @@ default_azure_credentials <- function(api_key = NULL, token = NULL) {
     return(function() list(Authorization = paste("Bearer", token)))
   }
 
+  azure_openai_scope <- "https://cognitiveservices.azure.com/.default"
+
+  # Detect viewer-based credentials from Posit Connect.
+  if (
+    is_installed("connectcreds") &&
```

Comment:
Maybe wrap this in a ellmer utility function? (I really don’t like the line breaks)